zeronote/docker-compose.yml

```
@@ -0,0 +1,14 @@
+version: "3.7"
+services:
+  app_proxy:
+    environment:
+      APP_HOST: zeronote
```

Use full service name.
```suggestion
      APP_HOST: zeronote_app_1
```